test/visualTest.ts

```diff
@@ -123,56 +125,30 @@ describe("DualKpi", () => {
         });
     });
 
-    describe("Capabilities tests", () => {
```

Comment:
Why removing this test?

Reply:
This test checks that each property has `displayNameKey` in capabilities.json but after migrating to new formatting model we specify `displayNameKey` in formatting model settings. I didn't want to duplicate the keys so I removed them from capabilities.json, making it thinner and easier to read.